Makefile

```diff
@@ -0,0 +1,31 @@
+VERSION=$(shell git describe --tags --dirty --always)
+
+.PHONY: build
+build:
+	go build -ldflags "-X 'github.com/conduitio/conduit-connector-connectorname.version=${VERSION}'" -o conduit-connector-connectorname cmd/connector/main.go
+
+.PHONY: test
+test:
+	go test $(GOTEST_FLAGS) -race ./...
+
+.PHONY: test-integration
+test-integration:
+	# run required docker containers, execute integration tests, stop containers after tests
+	docker compose -f test/docker-compose.yml up -d
+	go test $(GOTEST_FLAGS) -v -race ./...; ret=$$?; \
+		docker compose -f test/docker-compose.yml down; \
+		exit $$ret
```

Comment:
This is redundant, we don't have a docker compose file for tests in this connector.